exercises/spiral-matrix/example.js

```diff
@@ -0,0 +1,27 @@
+class SpiralMatrix {
+
+  static ofSize(size) {
+    const spiral = [];
+    for (; spiral.length < size; spiral.push([]));
```

Comment:
Take a look at [`Array.prototype.fill`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/fill). I think there should be a more clear way of creating a matrix, kind of `new Array(n).fill(new Array(m)` or so.